apps/docs/src/app/platform/component-docs/platform-table/platform-table-examples/platform-table-no-items-template-example.component.html

```diff
@@ -0,0 +1,65 @@
+<fdp-table
+    [dataSource]="source"
+    contentDensity="compact"
+>
+
+    <fdp-table-toolbar
+        title="Order Line Items"
+        [hideItemCount]="false"
+    >
+        <fdp-table-toolbar-actions>
+            <fdp-button
+                label="Action One"
+                contentDensity="compact"
+                (click)="alert('Action One')"
+            ></fdp-button>
+            <fdp-button
+                label="Action Two"
+                contentDensity="compact"
+                (click)="alert('Action Two')"
+            ></fdp-button>
+        </fdp-table-toolbar-actions>
+    </fdp-table-toolbar>
+
+    <fdp-column
+        name="name"
+        key="name"
+        label="Name"
+        align="start"
+    >
+    </fdp-column>
+
+    <fdp-column
+        name="description"
+        key="description"
+        label="Description"
+    >
+    </fdp-column>
+
+    <fdp-column
+        name="price"
+        key="price.value"
+        label="Price"
+        align="end"
+    >
+    </fdp-column>
+
+    <fdp-column
+        name="status"
+        key="status"
+        label="Status"
+        align="center"
+    >
+    </fdp-column>
+
+    <fdp-table-no-data-wrapper>
+        <div style="width: 100%; display: flex; justify-content: center;">
+            <figure fd-illustrated-message [svgConfig]="sceneConfig">
+                <figcaption fd-illustrated-message-figcaption>
+                    <h3 fd-illustrated-message-title>No matching items found.</h3>
+                    <p fd-illustrated-message-text>Check the filter settings.</p>
+                </figcaption>
+            </figure>
+        </div>
+    </fdp-table-no-data-wrapper>
```

Comment:
Well, as for me it would be more convenient for users to create more high level component. Which by default set flex-box styles and Inputs like `position`, `icon`, etc. 
Because usual such component uses for many types of tables in app, and for every table users need to copy-paste huge piece of code or create own component, I think it's extra efforts for users. 

But if dxp team expects current implementation - ok.

Reply:
I think in the future we can add platform illustrated message if needed. For this case we just need to bring ability to set custom component as a no content message. I've simplified flex and centering so that user don't have to provide such styling by himself